hc/api/admin.py

```diff
@@ -162,6 +162,8 @@ def formatted_kind(self, obj):
             return "Pushbullet"
         elif obj.kind == "po":
             return "Pushover"
+        elif obj.kind == "aft":
```

Comment:
what does `aft` signify? could it be more descriptive

Reply:
So remember what I said on the channel, for any new variable, key etc that you add ensure it is descriptive enough. That should be the guiding standard not what is already on the codebase.